deepfence_server/model/scans.go

```diff
@@ -65,6 +65,7 @@ type ScanTriggerCommon struct {
 type NodeIdentifier struct {
 	NodeID   string `json:"node_id" required:"true"`
 	NodeType string `json:"node_type" required:"true" enum:"image,host,container,cloud_account,cluster,registry,pod"`
+	NodeName string `json:"node_name" required:"false"`
 }
 
```

Comment:
Now I am not sure we should do that..
Can't we run two queries:
- One with the NodeName as condition to find all the NodeIDs
- The next query will do the actual search using those nodeIDs